src/toil/fileStores/abstractFileStore.py

```diff
@@ -296,7 +299,8 @@ def readGlobalFile(self, fileStoreID, userPath=None, cache=True, mutable=False,
         Makes the file associated with fileStoreID available locally. If mutable is True,
         then a copy of the file will be created locally so that the original is not modified
         and does not change the file for other jobs. If mutable is False, then a link can
-        be created to the file, saving disk resources.
+        be created to the file, saving disk resources. The file that is downloaded will be 
+        executable if it was originally uploaded from an executable file on the local filesystem.
```

Comment:
Since I'm also insisting that non-executable files stay non-executable, maybe we should write "if and only if" here and in the import/export docs.